kusto/ingest/ingest.go

```diff
@@ -124,47 +123,49 @@ type DataFormat = properties.DataFormat
 // to allow properties and ingest to both import without a cycle.
 const (
 	// DFUnknown indicates the EncodingType is not set.
-	DFUnknown DataFormat = 0
-	// CSV indicates the source is encoded in comma seperated values.
-	CSV DataFormat = 1
-	// JSON indicates the source is encoded in Javscript Object Notation.
-	JSON DataFormat = 2
+	DFUnknown DataFormat = properties.DFUnknown
 	// AVRO indicates the source is encoded in Apache Avro format.
-	AVRO DataFormat = 3
-	// Parquet indicates the source is encoded in Apache Parquet format.
-	Parquet DataFormat = 4
+	AVRO DataFormat = properties.AVRO
+	// ApacheAVRO indicates the source is encoded in Apache avro2json format.
+	ApacheAVRO DataFormat = properties.ApacheAVRO
+	// CSV indicates the source is encoded in comma seperated values.
+	CSV DataFormat = properties.CSV
+	// JSON indicates the source is encoded as one or more lines, each containing a record in Javscript Object Notation.
+	JSON DataFormat = properties.JSON
+	// MultiJSON indicates the source is encoded in JSON-Array of individual records in Javscript Object Notation.
+	MultiJSON DataFormat = properties.MultiJSON
 	// ORC indicates the source is encoded in Apache Optimized Row Columnar format.
-	ORC DataFormat = 5
+	ORC DataFormat = properties.ORC
+	// Parquet indicates the source is encoded in Apache Parquet format.
+	Parquet DataFormat = properties.Parquet
 	// PSV is pipe "|" separated values.
-	PSV DataFormat = 6
+	PSV DataFormat = properties.PSV
 	// Raw is a text file that has only a single string value.
-	Raw DataFormat = 7
+	Raw DataFormat = properties.Raw
 	// SCSV is a file containing semicolon ";" separated values.
-	SCSV DataFormat = 8
+	SCSV DataFormat = properties.SCSV
 	// SOHSV is a file containing SOH-separated values(ASCII codepont 1).
-	SOHSV DataFormat = 9
-	// TSV is a file containing table seperated values ("\t").
-	TSV DataFormat = 10
+	SOHSV DataFormat = properties.SOHSV
+	// SStream indicats the source is encoded as a Microsoft Cosmos Structured Streams format
+	SStream DataFormat = properties.SStream
```

Comment:
Please remove as this is MS internal according to comments

Reply:
We discussed this internally. 
Although Go is not commonly used in MS, the SDK serves both MS and non MS develpments and therefore should support all formats.